pulsar-client-tools/src/main/java/com/yahoo/pulsar/admin/cli/CmdBrokers.java

```diff
@@ -48,9 +48,23 @@ void run() throws Exception {
         }
     }
 
+    @Parameters(commandDescription = "Update dynamic-serviceConfiguration of broker")
```

Comment:
added both commands:
- getUpdatableConfigName
- getAllConfigurations to get key,value for all configs